qgsquick/from_qgis/plugin/editor/qgsquickrange.qml

```diff
@@ -173,8 +203,18 @@ Item {
         width: slider.availableWidth
         height: implicitHeight
         radius: 2 * QgsQuick.Utils.dp
-        color:  fieldItem.enabled ? customStyle.fields.fontColor : customStyle.fields.backgroundColorInactive
+        color: fieldItem.enabled ? customStyle.fields.fontColor : customStyle.fields.backgroundColorInactive
       }
+
+      handle: Rectangle {
+              x: slider.leftPadding + slider.visualPosition * (slider.availableWidth - width)
+              y: slider.topPadding + slider.availableHeight / 2 - height / 2
+              implicitWidth: slider.height * 0.5
+              implicitHeight: implicitWidth
+              radius: height * 0.5
+              color: "white"
+              border.color: customStyle.fields.backgroundColorInactive
```

Comment:
I would maybe remove border on the handle so that it looks similar to switch widget. What do you think?
![image](https://user-images.githubusercontent.com/22449698/110323049-50e7c580-8014-11eb-9eb2-276d787f232b.png)